prisma/seed.ts

```diff
@@ -23,6 +25,118 @@ async function seed() {
     },
   });
 
+  await prisma.record.createMany({
+    data: [
+      // A record
+      {
+        id: 1,
+        username: username,
+        name: 'OSD700',
+        type: `A`,
+        value: '172.5.251.0',
+        description: 'For final porject.',
+        course: 'OSD700',
+        ports: '8080, 1234',
+        createdAt: new Date('2023-06-15T01:01:01'),
+        expiresAt: new Date('2023-09-15T12:59:59'),
+        status: `active`,
+      },
+      // AAAA record
+      {
+        id: 2,
+        username: username,
+        name: 'WEB244',
+        type: `AAAA`,
+        value: '2002:db8:1111:4444:5555:6666:7777:8888',
+        description: 'Assignment 1',
+        course: 'WEB244',
+        ports: '8080',
+        createdAt: new Date('2023-06-15T01:01:01'),
+        expiresAt: new Date('2023-08-15T12:59:59'),
+        status: `active`,
+      },
+      // CNAME record
+      {
+        id: 3,
+        username: username,
+        name: 'WEB466AS2',
+        type: `CNAME`,
+        value: 'some.external.domain',
+        description: 'Assignment 2',
+        course: 'WEB466',
+        ports: '8000',
+        createdAt: new Date('2023-06-15T01:01:01'),
+        expiresAt: new Date('2023-10-20T12:59:59'),
+        status: `active`,
+      },
+      // TXT record
+      {
+        id: 4,
+        username: username,
+        name: 'CCP522',
+        type: `TXT`,
+        value: 'google-site-verification=6P08Ow5E-8Q0m6vQ7FMAqAYIDprkVV8fUf_7hZ4Qvc8',
+        createdAt: new Date('2023-06-15T01:01:01'),
+        expiresAt: new Date('2023-07-03T12:59:59'),
+        status: `active`,
+      },
+      // expired record with error status
+      {
+        id: 5,
+        username: username,
+        name: 'expired',
+        type: `AAAA`,
+        value: '2002:db8:1111:4444:5555:6666:7777:8888',
+        createdAt: new Date('2022-06-15T01:01:01'),
+        expiresAt: new Date('2022-09-15T12:59:59'),
+        status: `error`,
+      },
+      // a pending record
+      {
+        id: 6,
+        username: username,
+        name: 'pending',
+        type: `AAAA`,
+        value: '2002:db8:1111:4444:5555:6666:7777:8888',
+        createdAt: new Date('2022-03-15T01:01:01'),
+        expiresAt: new Date('2022-09-15T12:59:59'),
+        status: `pending`,
+      },
+    ],
+  });
+
+  await prisma.certificate.create({
+    data: {
+      id: 1,
+      username: username,
+      subject: `CN=*.${username}.example.com`,
+      certificate:
+        '-----BEGIN CERTIFICATE-----ApfFCv0O65TYkp5jEWSlU8PhKYD43nXA=-----END CERTIFICATE-----',
+      orderUrl: `orderURL.example.com`,
+      privkey:
```

Comment:
Maybe follow camelCase convention here to be consistent with the rest of the code.

Reply:
OK, done.